starter-code/scripts/controllers/aboutController.js

```diff
@@ -4,7 +4,12 @@
   var aboutController = {};
 
   aboutController.reveal = function() {
-    /* TODO: Use your DOM skills to reveal only the about section! */
+    /* TODO: DONE! Use your DOM skills to reveal only the about section! */
+    $('.main-nav').on('click', '.tab', function(e) {
+      e.preventDefault();
+      $('.tab-content').hide();
+      $('#' + $(this).data('content')).fadeIn();
+    });
```

Comment:
There is no need to make this into an event listener! Page.js takes care of that for your and "intercepts" a request when a link is clicked (with a specified URL), calling a callback function (this method) for you. Therefore, all you have to do is define what you want to happen when the user clicks on that link (you don't need to create an event). So, here, inside of this method, you just need to hide all of the sections except for the appropriate one (in this controller, it would be the about-me section).